CSharpMath/Atom/LaTeXSettings.cs

```diff
@@ -469,11 +471,11 @@ public static class LaTeXSettings {
         { @"\threeunderdot", new Accent("\u20E8") },
         { @"\TeX", new Inner(Boundary.Empty, new MathList(
             new Variable("T") { FontStyle = FontStyle.Roman },
-            new Space(-1/6f * Structures.Space.EmWidth) { FontStyle = FontStyle.Roman },
-            new RaiseBox(-1/2f * Structures.Space.ExHeight,
+            new Space(-1 / 6f * Structures.Space.EmWidth) { FontStyle = FontStyle.Roman },
+            new RaiseBox(-1 / 2f * Structures.Space.ExHeight,
               new MathList(new Variable("E") { FontStyle = FontStyle.Roman })
             ) { FontStyle = FontStyle.Roman },
-            new Space(-1/8f * Structures.Space.EmWidth) { FontStyle = FontStyle.Roman },
+            new Space(-1 / 8f * Structures.Space.EmWidth) { FontStyle = FontStyle.Roman },
```

Comment:
I am going for a fraction-like style instead of looking like division.

Reply:
I will revert add and fix then. This is all VS auto-formatting.